docs/user/filesystem.md

```diff
@@ -195,29 +195,283 @@ The contents of multiple pools/containers will be accessible via this
+dfuse           537G  5.1G  532G   1% /scratch_fs/dfuse
+$
 ```
 
 ### Launching via fstab
 
 DFuse can be launched via fstab and the standard mount command, it will parse -o options
 and extract pool=<name>,container=<name> if provided and ignore any other filesystem options given.
 
+There are few use cases described below to explain how systemd or /etc/fstab can be used to mount
+the daos container using dfuse.
+
+#### 1> User can mount the file system using mount.fuse3 command
+
+```bash
+$  dmg pool create --scm-size=8G --nvme-size=64G --label=samirrav_pool -u samirrav@
+Creating DAOS pool with manual per-engine storage allocation: 8.0 GB SCM, 64 GB NVMe (12.50% ratio)
+Pool created with 11.11%,88.89% storage tier ratio
+--------------------------------------------------
+  UUID                 : b43b06fe-4013-4177-911c-6d230b88fe6e
+  Service Ranks        : [1-5]
+  Storage Ranks        : [0-7]
+  Total Size           : 576 GB
+  Storage tier 0 (SCM) : 64 GB (8.0 GB / rank)
+  Storage tier 1 (NVMe): 512 GB (64 GB / rank)
+
+$  daos cont create samirrav_pool samirrav_cont --type=POSIX
+  Container UUID : 6efdc02c-5eaa-4a29-a34b-a062f1fe3371
+  Container Label: samirrav_cont
+  Container Type : POSIX
+
+Successfully created container 6efdc02c-5eaa-4a29-a34b-a062f1fe3371
+$  daos cont get-prop samirrav_pool samirrav_cont
+Properties for container samirrav_cont
+Name                                    Value
+----                                    -----
+Highest Allocated OID                   0
+Checksum                                off
+Checksum Chunk Size                     32 KiB
+Compression                             off
+Deduplication                           off
+Dedupe Threshold                        4.0 KiB
+EC Cell Size                            64 KiB
+Performance domain affinity level of EC 1
+Encryption                              off
+Global Version                          2
+Group                                   samirrav@
+Label                                   samirrav_cont
+Layout Type                             POSIX (1)
+Layout Version                          1
+Max Snapshot                            0
+Owner                                   samirrav@
+Redundancy Factor                       rf0
+Redundancy Level                        node (2)
+Performance domain affinity level of RP 3
+Server Checksumming                     off
+Health                                  HEALTHY
+Access Control List                     A::OWNER@:rwdtTaAo, A:G:GROUP@:rwtT
+$  mkdir /scratch_fs/daos_dfuse_samir
+$  mount.fuse3 dfuse /scratch_fs/daos_dfuse_samir -o pool=samirrav_pool,container=samirrav_cont
+$  touch /scratch_fs/daos_dfuse_samir/foo
+$  ls -l /scratch_fs/daos_dfuse_samir/
+total 0
+-rw-rw-r-- 1 samirrav samirrav 0 Sep 23 15:49 foo
+$  df -h | grep fuse
+dfuse                         537G  5.1G  532G   1% /scratch_fs/daos_dfuse_samir
+$
+```
+
+#### 2> Use dfuse entry in /etc/fstab to mount the container.
```

Comment:
This requires to be root, right? Maybe this should be clarified for each use case.